test/integration/server.js

```diff
@@ -2146,6 +2146,20 @@ describe('Wallet service', function() {
         });
       });
     });
+    it('should not use UTXO provided in utxosToExclude option', function(done) {
```

Comment:
This test assumes a specific utxo selection algorithm (in particular, the fact that utxos are selected in the order they are retrieved from the block explorer). Maybe a test where the txp cannot be created because non-excluded utxos are insufficient (but enough utxos exist to create the txp) would be nice?